### PR TITLE
i18n(ko-KR): update `configuration-reference.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- update `configuration-reference.mdx`
- #6109
- #6156 

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/ko/reference/configuration-reference.mdx [hunk 1]
```diff
@@ -564,7 +564,8 @@ prefetch: {
 - `'tap'`: 링크를 클릭하기 직전에 미리 가져옵니다.
 - `'hover'`: 링크 위로 마우스를 가져가거나 링크에 초점을 맞추면 미리 가져옵니다. (기본값)
 - `'viewport'`: 링크가 뷰포트에 들어갈 때 미리 가져옵니다.
-- `'load'`: 페이지가 로드된 후에 페이지의 모든 링크를 미리 가져오세요.。
+- `'load'`: 아무런 제한 없이 링크를 미리 가져옵니다.
```
Comment (sarah11918) [inline src/content/docs/ko/reference/configuration-reference.mdx:567]: @jsparkdev Can you please check this to #6156 -- I think the existing text is the corrected one!

Comment (jsparkdev) [reply src/content/docs/ko/reference/configuration-reference.mdx:567]: After I edit this part, I will make a new commit and let you know!

Comment (jsparkdev) [reply src/content/docs/ko/reference/configuration-reference.mdx]: The content has been updated and committed.

**Review by sarah11918 — APPROVED** [review]
Thank you for keeping up!

**Pull request merged**